packages/osd-std/package.json

```diff
@@ -1,20 +1,22 @@
 {
   "name": "@osd/std",
-  "main": "./target/index.js",
-  "types": "./target/index.d.ts",
+  "browser": "./target/web/index.js",
+  "main": "./target/node/index.js",
+  "types": "./target/types/index.d.ts",
   "version": "1.0.0",
   "license": "Apache-2.0",
   "private": true,
-  "scripts": {
-    "build": "tsc",
-    "osd:bootstrap": "yarn build"
-  },
+  "sideEffects": false,
```

Comment:
I'm not familiar - what does this do?

Reply:
It tells webpack that the content of this package can safely be tree-shaken.

Reply:
excellent, thanks@